core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -274,7 +274,10 @@ object ConsumerGroupCommand extends Logging {
 
     protected def opts: ConsumerGroupCommandOptions
 
-    protected def getLogEndOffset(topicPartition: TopicPartition): LogOffsetResult
+    protected def getLogEndOffset(topicPartition: TopicPartition): LogOffsetResult =
```

Comment:
It seems this method is still used in a loop in line 706 and line 771. Should we also replace them with batch call?

Reply:
You're right. I'll try to consolidate this in the next commit. Thanks for catching it.